src/main/java/com/github/jmchilton/blend4j/galaxy/WorkflowsClient.java

```diff
@@ -22,4 +22,11 @@
   ClientResponse runWorkflowResponse(WorkflowInputs workflowInputs);
 
   WorkflowOutputs runWorkflow(WorkflowInputs workflowInputs);
+  
+  /**
+   * Deletes the workflow with the given id.  This action is irreversible.
+   * @param id The id of the workflow to delete.
+   * @return A {@link ClientResponse} for this request.
+   */
+  ClientResponse deleteWorkflowResponse(String id);
```

Comment:
The JSON response I get from deleting a workflow is:

```json
{
    "annotation": null, 
    "deleted": true, 
    "id": "417e33144b294c21", 
    "inputs": {
        "48": {
            "label": "WorkflowInput1", 
            "value": ""
        }, 
        "49": {
            "label": "WorkflowInput2", 
            "value": ""
        }
    },
...
```

That is, it the workflow itself with the field **deleted** set to true.

Mostly the other two types only return a `ClientResponse` because I didn't want to spend the time writing new objects in blend4j to represent the responses (so, laziness really).  You think it would be a good idea to write them?